src/components/Section/Citizenship/Citizenship.jsx

```diff
@@ -1,198 +1,98 @@
-            />
+  render() {
+    const { subsection = 'intro', formType } = this.props
+    const subsectionClasses = `view view-${subsection || 'unknown'}`
+    const isReview = subsection === 'review'
+    const title = isReview && i18n.t('review.title')
+    const para = isReview && i18n.m('review.para')
 
-            <hr className="section-divider" />
-            <Passports
-              name="passports"
-              {...this.props.Passports}
-              section="citizenship"
-              subsection="passports"
-              dispatch={this.props.dispatch}
-              onUpdate={this.handleUpdate.bind(this, 'Passports')}
-              onError={this.handleError}
-              required={true}
-              scrollIntoView={false}
-            />
-          </SectionView>
+    return (
+      <div className="section-view">
+        {title && <h1 className="title">{title}</h1>}
+        {para}
 
-          <SectionView
-            name="status"
-            back="citizenship/intro"
-            backLabel={i18n.t('citizenship.destination.intro')}
-            next="citizenship/multiple"
-            nextLabel={i18n.t('citizenship.destination.multiple')}>
-            <Status
-              name="status"
-              {...this.props.Status}
-              dispatch={this.props.dispatch}
-              onUpdate={this.handleUpdate.bind(this, 'Status')}
-              onError={this.handleError}
-            />
-          </SectionView>
+        <div className={subsectionClasses}>
+          {isReview && (
+            <div className="top-btns"><ErrorList /></div>
+          )}
 
-          <SectionView
-            name="multiple"
-            back="citizenship/status"
-            backLabel={i18n.t('citizenship.destination.status')}
-            next="citizenship/passports"
-            nextLabel={i18n.t('citizenship.destination.passports')}>
-            <Multiple
-              name="multiple"
-              {...this.props.Multiple}
-              dispatch={this.props.dispatch}
-              onUpdate={this.handleUpdate.bind(this, 'Multiple')}
-              onError={this.handleError}
-              scrollToBottom={this.props.scrollToBottom}
-            />
-          </SectionView>
+          {this.getCitizenshipSubsections()}
 
-          <SectionView
-            name="passports"
-            back="citizenship/multiple"
-            backLabel={i18n.t('citizenship.destination.multiple')}
-            next="citizenship/review"
-            nextLabel={i18n.t('citizenship.destination.review')}>
-            <Passports
-              name="passports"
-              {...this.props.Passports}
-              dispatch={this.props.dispatch}
-              onUpdate={this.handleUpdate.bind(this, 'Passports')}
-              onError={this.handleError}
-              scrollToBottom={this.props.scrollToBottom}
-            />
-          </SectionView>
-        </SectionViews>
+          <SectionNavigation
+            section={CITIZENSHIP}
```

Comment:
In previous sections, we imported all the sections. Since that's unnecessary, I only imported the Citizenship section.